jmh-core/src/test/java/org/openjdk/jmh/profile/PerfParseTest.java

```diff
@@ -62,26 +62,4 @@ public void parseRaggedSymbols() {
         }
     }
 
-    @Test
```

Comment:
Please don't remove this test. It should verify that it parses the proper event names, now with event modifiers. This would verify that `:` are parsed/stripped correctly.

Reply:
Sure, I've added the test back.